earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +51,13 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+fusiontestdir = os.path.join(testdir, 'fusion/testdata')
+shutil.rmtree(fusiontestdir, ignore_errors=True)
+testdatasrc = Dir('../../../fusion/testdata').abspath
+shutil.copytree(testdatasrc, fusiontestdir)
```

Comment:
Done. I actually prefer this method because it makes sure that it's executed at the correct build stage, unlike shutil which is executed immediately.